packages/SwingSet/test/workers/test-worker.js

```diff
@@ -28,7 +28,7 @@ test('local vat manager', async t => {
   t.deepEqual(JSON.parse(c.kpResolution(c.bootstrapResult).body), expected);
 });
 
-test('xs vat manager', async t => {
+test.skip('xs vat manager', async t => {
```

Comment:
that's fine, this test is going away in @dckc 's work anyways, to be replaced with the `xsnap` variant

Reply:
I don't know why I thought this was true. It's not. The `xsnap` variant uses the same test.